### ISSUE TITLE
Question [theoretical]  about the possibility of automating some commands Photopea

### ISSUE THREAD

**Issue by Pt-home** [photopea/photopea#5650]
Hello!
Photopea now has a few excellent, without exaggeration, tools without direct analogies in PS functionality.

Question: 
Reduce Colors, Vectorize Bitmap, Remove BG and Magic Cut (with default parametrs) - in theory, now possible to launch them through the formation of a descriptor for executeAction? If "yes", then - how?:)

Its commands are not added to the Action (this feature made it easy to automate the "exclusive" Photopea filters: kaleidoscope and company).
I have already "dug up and down" pp.js but didn't find an answer:)
Thanks for help!

**Comment by photopea** [comment]
I think these features need different parameters for each image, so you will not get a good result by using the same parameters for many images. Also, if someone offers your features in their own tool (through our API), the users dont watch our ads, and we make no money.

The "Remove BG" feature - we pay several cents for each call, so anyone could make a script "Make Photopea X dollars poorer", when you enter a number X and it performs a certain number of calls of "Remove BG" in a loop.

**Comment by Pt-home** [comment]
Ok, your arguments are very convincing. About Remuv BG, I did not know that he uses a third-party neural network... 
The only thing is about Magits Cut. Having a GIF of 30-300 frames, the background on each is often removed by the default setting, and here automation could often be useful... But if it uses a third-party neural network too - question closed. Thanks

**Comment by photopea** [comment]
Cant you remove the GIF background with an action with Color Range or a Magic Wand?

**Comment by Pt-home** [comment]
You are absolutely right, and it is good that some things can be done in different ways, some easier, some more difficult. It's worth checking the most obvious way first. Thanks for the replies and sorry for wasting valuable time on them

**Comment by photopea** [comment]
So I would rather not offer these features of Photopea as Scripts or Actions for now.
